RecoLocalCalo/EcalRecProducers/plugins/alpaka/TimeComputationKernels.h

```diff
@@ -755,10 +755,10 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::ecal::multifit {
           continue;
 
         int const inputGtx = ch >= offsetForInputs ? gtx - offsetForInputs * nsamples : gtx;
-        auto const* digis = ch >= offsetForInputs ? digisDevEE.data()->data() : digisDevEB.data()->data();
+        auto const digis = ch >= offsetForInputs ? digisDevEE.data() : digisDevEB.data();
 
-        auto const gainIdPrev = ecalMGPA::gainId(digis[inputGtx - 1]);
-        auto const gainIdNext = ecalMGPA::gainId(digis[inputGtx]);
+        auto const gainIdPrev = ecalMGPA::gainId(digis[inputGtx - 1][0]);
+        auto const gainIdNext = ecalMGPA::gainId(digis[inputGtx][0]);
```

Comment:
@thomreis this, too 🤷🏻 

Shouldn't it be
```c++
        auto const gainIdPrev = ecalMGPA::gainId(digis[0][inputGtx - 1]);
        auto const gainIdNext = ecalMGPA::gainId(digis[0][inputGtx]);
```
?

But then, why do we use only the first element of `digis` ?

Reply:
I think the original code is actually what is wanted here. Just as for the comment above.